index.html

```diff
@@ -42,10 +42,10 @@
       <h3>I care about sharing improvements.</h3>
     </a>
     <p>
-      The GPL (<a href="licenses/gpl-2.0/">V2</a> or <a href="licenses/gpl-3.0/">V3</a>) is a copyleft license that requires anyone who distributes your code or a derivative work to make the source available under the same terms. V3 is similar to V2, but further restricts use in hardware that forbids software alterations.
+      The <a href="licenses/gpl-3.0/">GNU GPL</a> is a copyleft license that lets people do anything they want with your code as long as they give the same freedoms to anyone   else to which give your code or something based on it.
```

Comment:
Ooops, I didn't mean to commit this change yet. I'll revert and open another PR with suggested wording changes for GPL.